graphs/models.py

```diff
@@ -310,6 +310,16 @@ class Edge(Base):
             ForeignKeyConstraint([user_id, graph_id, tail_node_id], [Node.user_id, Node.graph_id, Node.node_id], ondelete="CASCADE", onupdate="CASCADE"), {})
     #no relationship specified
 
+class share_graph_event(Base):
+    __tablename__ = 'share_graph_event'
```

Comment:
Need some docs on the schema. What does the table store? Why do you need each attribute?